### DISCUSSION TITLE
Debug / Sample code in Production

### DISCUSSION THREAD

**Discussion by Nicole4Fox** [astarub/campus_app#129] [General]
Is there any reason to leave debug code, print statements and not yet implemented classes in the main branch? As far as im aware there should be a development/debug branch and there should be a branch without such codes. Thats exactly what branches are for. Id like to see debug and experimental or not yet implemented code to be removed from the main and deploy branches to keep the codebase cleaner. Is there any good reason not to do this?

**Comment by henry-herrmann** [comment]
There is no reason, will be removed with the next release.
